test/FastANI_server_test.py

```diff
@@ -107,21 +107,17 @@ def test_fastani_binary(self):
 
     def test_run_fast_ani(self):
         """
-        Test a basic call to FastANIImpl#fast_ani using a query and reference assembly
-        Copy the FastANI example data into the scratch dir
+        Test a run of the full FastANIImpl function using several assembly/genome refs.
         """
-        a_path = os.path.join(self.scratch, 'a.fna')
-        b_path = os.path.join(self.scratch, 'b.fna')
-        data_dir = os.path.join(os.path.dirname(__file__), 'data')
-        copyfile(os.path.join(data_dir, 'shigella.fna'), a_path)
-        copyfile(os.path.join(data_dir, 'ecoli.fna'), b_path)
-        a_ref = self.load_fasta_file(a_path, 'test_assembly_a')
-        b_ref = self.load_fasta_file(b_path, 'test_assembly_b')
-        refs = [a_ref, b_ref]
+        assembly_id1 = '34819/10/1'
```

Comment:
Using public CI data because it's such a pain to upload everything over and over for tests.